generated/nifgen/session.py

```diff
@@ -853,6 +843,16 @@ class _SessionBase(object):
 
     Returns a brief description of NI-FGEN.
     '''
+    major_version = _attributes.AttributeViInt32(1050503)
```

Comment:
Do you know why this shown up in a different position? (not that it matter much)

Reply:
Because the actual attribute name is not `NIFGEN_ATTR_MAJOR_VERSION`, it is actually `NIFGEN_ATTR_SPECIFIC_DRIVER_MAJOR_VERSION`. But to not break backwards compatibility I had to keep the old name, but it is alphabetized by the actual name.